src/System.Runtime.Extensions/src/System/AppDomain.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Internal.Runtime.Augments;
+using System;
+using System.Reflection;
+
+namespace System
+{
+    public partial class AppDomain
+    {
+        private static AppDomain domain = new AppDomain();
+        private AppDomain() {}
+        public static AppDomain CurrentDomain => domain;
+        public Assembly[] GetAssemblies() => AppDomainAugments.GetAssemblies();
+        public string BaseDirectory => "";
+        public string RelativeSearchPath => "";
```

Comment:
Are empty strings the right thing to return here?  What do folks normally use these values for?